packages/host/app/components/operator-mode/create-file-modal.gts

```diff
@@ -93,37 +100,39 @@ export default class CreateFileModal extends Component<Signature> {
                 @onSelect={{this.onSelectRealm}}
               />
             </FieldContainer>
-            <FieldContainer
-              @label={{if
-                (eq this.maybeFileType.id 'card-instance')
-                'Adopted From'
-                'Inherits From'
-              }}
-              class='field'
-              data-test-inherits-from-field
-            >
-              <div class='field-contents'>
-                {{#if this.definitionClass}}
-                  <Pill @inert={{true}}>
-                    {{this.definitionClass.displayName}}
-                  </Pill>
-                {{else}}
-                  {{#if this.selectedCatalogEntry}}
-                    <SelectedTypePill @entry={{this.selectedCatalogEntry}} />
+            {{#unless (eq this.fileType.id 'duplicate-instance')}}
+              <FieldContainer
+                @label={{if
+                  (eq this.maybeFileType.id 'card-instance')
+                  'Adopted From'
+                  'Inherits From'
+                }}
+                class='field'
+                data-test-inherits-from-field
+              >
+                <div class='field-contents'>
+                  {{#if this.definitionClass}}
+                    <Pill @inert={{true}}>
+                      {{this.definitionClass.displayName}}
+                    </Pill>
+                  {{else}}
+                    {{#if this.selectedCatalogEntry}}
+                      <SelectedTypePill @entry={{this.selectedCatalogEntry}} />
+                    {{/if}}
+                    <Button
+                      class={{if this.selectedCatalogEntry 'change-trigger'}}
+                      @kind='text-only'
+                      @size='small'
+                      @disabled={{this.isCreateRunning}}
+                      {{on 'click' (perform this.chooseType)}}
+                      data-test-select-card-type
+                    >
+                      {{if this.selectedCatalogEntry 'Change' 'Select'}}
+                    </Button>
                   {{/if}}
-                  <Button
-                    class={{if this.selectedCatalogEntry 'change-trigger'}}
-                    @kind='text-only'
-                    @size='small'
-                    @disabled={{this.isCreateRunning}}
-                    {{on 'click' (perform this.chooseType)}}
-                    data-test-select-card-type
-                  >
-                    {{if this.selectedCatalogEntry 'Change' 'Select'}}
-                  </Button>
-                {{/if}}
-              </div>
-            </FieldContainer>
+                </div>
+              </FieldContainer>
+            {{/unless}}
```

Comment:
all that actually changed here was an `{{unless}}` that was added that wraps the original code.

Reply:
I appreciate this comment, makes for much easier reviewing, thanks. I’m surprised Github doesn’t make this clearer by now, or maybe it’s because it’s `.gts`.